.idea/modules.xml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
```

Comment:
This needs to be added to .gitignore as well.

Reply:
Done